src/D2L.Bmx/OktaAuthenticator.cs

```diff
@@ -191,11 +196,20 @@ async Task GetSessionCookieAsync() {
 						return;
 					}
 				}
-				var cookies = await page.GetCookiesAsync( orgBaseAddress ).WaitAsync( cancellationTokenSource.Token );
+				var cookies = await page.GetCookiesAsync( orgBaseAddress );
 				if( Array.Find( cookies, c => c.Name == "sid" )?.Value is string sid ) {
 					sessionIdTcs.SetResult( sid );
 				}
 			}
+			async Task GetOktaUserEmailAsync( IResponse response ) {
+				if( response.Url.Contains( $"{orgBaseAddress}enduser/api/v1/home" ) ) {
```

Comment:
we still shouldn't use enduser endpoints though.
the sessions/me endpoint gives you the same login name, right?

Reply:
I don't think it hurts to create an authenticated client first before verifying the username